src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.Helpers.cs

```diff
@@ -0,0 +1,171 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Text.Json.Serialization.Converters;
+
+namespace System.Text.Json.Serialization.Metadata
```

Comment:
nit: `namespace System.Text.Json.Serialization.Metadata;`

Reply:
Would be nice to save ourselves a bit of indentation, but I don't think we use the new namespace syntax anywhere in libraries at the moment. If we do, we should make sure it is applied consistently in a dedicated PR.

@stephentoub do we have any plans/concerns about doing that in libraries eventually?